### ISSUE TITLE
Make video bands optional

### ISSUE THREAD

**Issue by Luni-4** [bubblesub/bubblesub#48]
Rendering the video bands for 10 bit videos is pretty slow. Is it possible to create a command that enables or disables them at will? 

This new feature could also help people that wants to work only with audio and video, without having to use the bands.

**Comment by rr-** [comment]
I'd say this could be part of #18 since we manipulate other widgets there already.

**Comment by Luni-4** [comment]
That's true, but I prefer to keep it a separate issue because the audio splitter is composed both by spectrogram and video bands. It's not directly related to views

**Comment by rr-** [comment]
BTW I just pushed dfb2f2c and earlier a couple of other optimizations which should reduce UI lags with regard to spectrogram as well as video band processing.
Unfortunately I can't do miracles here because of the GIL.

**Comment by Luni-4** [comment]
> BTW I just pushed [dfb2f2c](https://github.com/rr-/bubblesub/commit/dfb2f2c892fb3eb7e60f088edac8555b2c6461f9) and earlier a couple of other optimizations which should reduce UI lags with regard to spectrogram as well as video band processing.
> Unfortunately I can't do miracles here because of the GIL, I think we're maxed out with what's possible short of using separate python processes for the workers, something that'd complicate a lot of things and I'd like to avoid if possible.

A further motive to make them optional then. In this way, if one doesn't have a powerful computer, he can use `bubblesub` the same.

**Comment by rr-** [comment]
Yes. However, rather than making this into a command, I'd put the boolean for it in the config. This would get read at the program startup.

If the user wants to play with it at runtime, I'd do it like vim, tmux, weechat and qutebrowser do - just have a `reload` command that re-reads the config. I'd avoid adding a command to change settings from the commandline, since bubblesub sometimes stores very complex objects in the config (hence yaml). Later on there could be a command that lets you fiddle with config options using a GUI at runtime. At any rate the bits from this paragraph belong to a future ticket or a wiki entry.

**Comment by Luni-4** [comment]
> Yes. However, rather than making this into a command, I'd put the boolean for it in the config. This would get read at the program startup.
> 
> If the user wants to play with it at runtime, I'd do it like vim, tmux, weechat and qutebrowser do - just have a `reload` command that re-reads the config. I'd avoid adding a command to change settings from the commandline, since bubblesub sometimes stores very complex objects in the config (hence yaml), and it's not obvious how to implement a `set` command that is able to play with it. 

I agree with this method completely.

> Because of this, later on there could be a command that lets you fiddle with config options using a GUI at runtime. At any rate the bits from this paragraph belong to a future ticket or a wiki entry.

That would be great, but it should be a low priority for me. Add this idea to the wiki if you want, thanks.

**Comment by Luni-4** [comment]
What about this one @rr-?
